### DISCUSSION TITLE
REHLDS RATES LAN

### DISCUSSION THREAD

**Discussion by lambdabest** [community/community#126622] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I'm looking for assistance with configuring the rate settings for a Counter-Strike 1.6 server. Specifically, I need to know the optimal rate settings for a server running on lan. Could you provide guidance on the best `rate`, `cl_updaterate`, and `cl_cmdrate`, sv_minrate, sv_maxrate and anothers settings for the latest version of reHLDS? Thank you!

ReHLDS Windows Server  ( 20.01.2024 )

**Comment by ebndev** [comment]
Hi @lambdabest,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a gaming community. You might want to try asking for help somewhere that focuses on this or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by ebndev** [comment]
Hi again @lambdabest!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
